### ISSUE TITLE
2nd subheader in table is read with first column's heading when using VoiceOver on iOS

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-core#1916]
# Bug Report

## Description
When shifting focus from content right before second header, "111-222-3333" to the second subheader in our [test subheader table](https://engineering.cerner.com/terra-core/#/tests/terra-table/table/table/table-with-subheaders), VO reads, "Name, Married, spans 3 columns, row 5, column 1". When shifting focus from "Dave Smith" to "Married" subheader, VO does not read, "Name" before reading the subheader content.

## Steps to Reproduce

1. Navigate to https://engineering.cerner.com/terra-core/#/tests/terra-table/table/table/table-with-subheaders
2. Turn on VoiceOver
3. Focus on phone number right before 2nd subheader.
4. Shift focus to 2nd subheader, "Married"

## Expected Behavior
VoiceOver should read subheader without prefixing table row header.

## Possible Solution
Not sure if there will be a clear way to resolve this or if this is more a VoiceOver bug. Need to look into accessible examples and see if there is a markup pattern for this type of concept (table subheaders).

## Environment

* Component Name and Version: terra-table
* Browser Name and Version: Mobile Safari 12 with VO on iOS

**Comment by bjankord** [comment]
I'm able to reproduce this issue with VoiceOver on iOS outside of terra using the same table markup: https://codepen.io/bjankord/pen/WBoBbm

I'm not sure there is a fix that we can implement to correct this. I believe this may be a bug with VoiceOver on iOS and how it handles this specific table markup. I've logged the following bug to webkit about this issue: https://bugs.webkit.org/show_bug.cgi?id=197886
